src/components/DayPicker.jsx

```diff
@@ -509,6 +517,12 @@ export default class DayPicker extends React.Component {
       translationValue: 0,
       nextFocusedDate: null,
       focusedDate: newFocusedDate,
+    }, () => {
+      // we don't want to focus on the relevant calendar day after a month transition
+      // if the user is navigating around using a mouse
+      if (this.state.withMouseInteractions && document.activeElement) {
```

Comment:
I'll add the check, although this is somewhat of an extreme edge case given the way the datepicker is managing focus. 

Basically, the `setState` that this is a callback of triggers a series of events that always ends in a `CalendarDay` being focused. So while it is possible that the user could blur between the `componentDidUpdate` call that focuses the CalendarDay in that leaf node and this setState callback 4 levels up in the tree, it's highly unlikely.